### PR TITLE
Possible bug with r-basics-with-tabular-data.md

### PR DESCRIPTION
**Pull request by drjwbaker**
I think this is wrong. For me l60..

> Take a look at the `AirPassengers` data set by typing `data(AirPassengers)` into the console

..doesn't work. What does work is just typing `AirPassengers`. Can you check this please @anisa-hawes before merging? (as it may just be my setup!)

### Checklist

- [x] Assign yourself in the "Assignees" menu
- [x] Assign at least one individual or team to "Reviewers"
  - [x] if the text needs to be translated, assign the relevant language team(s) as "Reviewers" and tag both the team as well as the managing edtor in your PR. Please follow the [translation request guidelines](https://github.com/programminghistorian/jekyll/wiki/Requesting-Translation-Guidelines) when writing your PR description
- [x] Add the appropriate "Label"
- [x] [Ensure the status checks pass](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#4-check-your-pr-status)
- [x] [Check the live preview of your PR on Netlify](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#5-preview-how-your-pr-looks-when-built-into-html)
- [ ] If this PR closes an open issue, add the phrase `Closes #ISSUENUMBER` to the description above

*If you are having difficulty fixing build errors, first consult <https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions> carefully, especially ["Common Build Errors"](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#common-build-errors). Then contact the technical team if you need further help.*

### CODE REVIEW

**Comment by DanielAlvesLABDH** [comment]
Olá @JimmyMedeiros82, nós traduzimos esta lição há pouco tempo. Será que temos o mesmo bug ou foi corrigido na tradução? Abraço

**Comment by anisa-hawes** [comment]
Thank you @drjwbaker! I will test it today, and will also fix the broken link which is causing the build to fail!

**Comment by anisa-hawes** [comment]
Thank you for noticing this error, @drjwbaker! I have tested this on my set up (I work on macOS BigSur v11.5.2) and found that the same was true. The command `data(AirPassengers)` didn't load the data set into my console. But, as you suggested, simply typing `AirPassengers` did!

Screenshot 2021-12-01 at 11 59 01
 
I can update this on EN, ES and PT.

Also: the link <https://joedicastro.com/pages/markdown.html> actually appears to be live. Perhaps the site was temporarily down when this build check ran...

**Comment by anisa-hawes** [comment]
Re-reading this, it seems to me that the whole paragraph is quite confusing. The sentences "Enter `data(AirPassengers)` into the console and hit *Enter*. On the next line, type in `AirPassengers` and hit *Enter* again" are redundant, effectively repeating the same instruction...

My suggestion is that this would be clearer:

Take a look at the AirPassengers data set. To load the data set into your console, type `AirPassengers` and hit Enter. This will bring up a table showing the number of passengers who flew on international airlines between January 1949 and December 1960, in thousands. You should see:

I'll also update the code blocks, so that the commands shown match those instructed in the lesson text.

**Comment by rivaquiroga** [comment]
Hi everyone! 
This is not really a bug. Both commands do different things.
`data(AirPassengers)` makes this built-in dataset available in your Global Environment. Running `AirPassengers` will only print it to the console. Doing the first step is not a requirement for the second one, BUT after loading a dataset with `data()` you will be able to see it in the Environment pane. Sometimes this is useful when teaching, especially if you are using an IDE like RStudio, because students can see the dataset as an object/variable. 

If you only print the object by calling it by its name, your RStudio session will look like this:

Captura de pantalla de 2021-12-01 09-54-44

If you run `data(AirPassengers)`, the dataset is loaded in your environment as a value

Captura de pantalla de 2021-12-01 09-54-23

If you print it with `AirPassengers` after that, you will also see information about the dataset in the environment. This will not happen if you skip the `data(AirPassengers)` step.

Captura de pantalla de 2021-12-01 09-57-31


Because the lesson is only using the R Console, it looks like `data(AirPassengers)` is not doing anything, but it is. 

This part of the lesson is a little confusing, because that command is not loading the dataset into the console, but into the global environment: _"Take a look at the AirPassengers data set by typing `data(AirPassengers)` into the console. This will load the data set into the console."_. 
If you decide to remove the `data(AirPassengers)` step, I would suggest using the word_print_ and not _load_ to refer to what happens when you just type `AirPassengers`.

**Comment by drjwbaker** [comment]
Thanks for your comments @rivaquiroga: shows how much I know about the R console! (not much).

>  If you decide to remove the `data(AirPassengers)` step, I would suggest using the word_print_ and not load to refer to what happens when you just type `AirPassengers`.

That makes the most sense to me as the `data(AirPassengers)` step appears redundant given how the lesson is written.

**Comment by jenniferisasi** [comment]
@rivaquiroga, who knows more R than I do, bit me to explain the "issue". Or misunderstanding of the function `data()` (loading) vs printing the dataset. 

Additionally, @anisa-hawes remember in my workshop I said you can Command-click on the dataset and it will open a spreadsheet-like view? This is why :) 

Re-reading the paragraph in my translation, I think it is clear but it's a bit messy so I would be happy to update it and explain it better

**Comment by anisa-hawes** [comment]
Thank you @rivaquiroga and @jenniferisasi!  

Taking what you have explained into account, I wonder if the following adjustment I've made to the EN wording might make best sense: 

First, you need to load the AirPassengers data set into your console. Type `data(AirPassengers)` and hit *Enter*[^1]. To view the data set, type in `AirPassengers` on the next line and hit *Enter* again. This will print a table showing the number of passengers who flew on international airlines between January 1949 and December 1960, in thousands. You should see:

> data()
> data(AirPassengers)
> AirPassengers

**Comment by anisa-hawes** [comment]
> That makes the most sense to me as the `data(AirPassengers)` step appears redundant given how the lesson is written.

Ah...  Let me know what you think is best, @drjwbaker.

**Comment by drjwbaker** [comment]
No no, go with what @rivaquiroga and @jenniferisasi say: they are the experts here, I hardly ever use rconsole. It feels like the article is using best practice for a reason, just hasn't quite explained why.

**Comment by rivaquiroga** [comment]
@anisa-hawes I would suggest for the first sentence: "First, you need to load the AirPassengers data set into your R session"

File: en/lessons/r-basics-with-tabular-data.md [hunk 1]
```diff
@@ -57,9 +57,10 @@ The console is where you will type commands. To clear the initial screen, go to
...
 
-Take a look at the `AirPassengers` data set by typing `data(AirPassengers)` into the console[^1]. This will load the data set into the console. To see the data set, type in `AirPassengers` on the next line and hit *Enter*. This will bring up a table showing the number of passengers who flew on international airlines between January 1949 and December 1960, in thousands. Enter `data(AirPassengers)` into the console and hit *Enter*. On the next line, type in `AirPassengers` and hit *Enter* again. You should see:
+First, you need to load the AirPassengers data set into your R session. Type `data(AirPassengers)` and hit *Enter*[^1]. To view the data set, type in `AirPassengers` on the next line and hit *Enter* again. This will print a table showing the number of passengers who flew on international airlines between January 1949 and December 1960, in thousands. You should see:
 
 ```
+> data()
```
Comment (rivaquiroga) [inline en/lessons/r-basics-with-tabular-data.md]: Maybe we can remove this extra `data()`, as it is mention in the previous paragraph just as an example of how to take a look to all the available datasets. It is not really a step expected for users to do for what comes next

Comment (anisa-hawes) [reply en/lessons/r-basics-with-tabular-data.md]: Yes, okay. I'll remove this!

**Comment by anisa-hawes** [comment]
Hello @JimmyMedeiros82 and @DanielAlvesLABDH. Might either of you have a moment to review paragraph 60 of nocoes-basicas-R-dados-tabulares.md, to ensure that the instructions are clear about what each command achieves?

`data(AirPassengers)` loads the data set into the R session
`AirPassengers` prints the data set 

The adjusted EN text is:

First, you need to load the AirPassengers data set into your R session. Type `data(AirPassengers)` and hit *Enter*[^1]. To view the data set, type in `AirPassengers` on the next line and hit *Enter* again. This will print a table showing the number of passengers who flew on international airlines between January 1949 and December 1960, in thousands. You should see:

**Review by DanielAlvesLABDH — DISMISSED** [review]

**Comment by anisa-hawes** [comment]
Thank you, @DanielAlvesLABDH!

Let us know if you have time to check the ES for clarity, @jenniferisasi or @rivaquiroga? When you're happy, I'll merge these changes.

**Comment by jenniferisasi** [comment]
I'll do it! Give me a sec...

**Comment by anisa-hawes** [comment]
Thank you, all!

There's a broken link now, so I will fix this.

**Comment by anisa-hawes** [comment]
Hmmm.... link seems to be correct & live....
<https://github.com/impresso/PH-passim-tutorial/blob/master/explore-passim-output.ipynb>
Will re-run...

**Review by anisa-hawes — APPROVED** [review]

**Pull request merged**